site/content/docs/user/ingress.md

```diff
@@ -57,6 +58,37 @@ nodes:
 EOF
 {{< /codeFromInline >}}
 
+### Ambassador
+
+[Ambassador](https://www.getambassador.io/) will be installed with the help of
+the [Ambassador operator](https://www.getambassador.io/docs/latest/topics/install/aes-operator/).
+
+First install the CRDs with
+
+{{< codeFromInline lang="bash" >}}
+kubectl apply -f https://github.com/datawire/ambassador-operator/releases/latest/download/ambassador-operator-crds.yaml
+{{< /codeFromInline >}}
+
+Now install the kind-specific manifest for installing Ambassador with the operator
+in the `ambassador` namespace:
+
+{{< codeFromInline lang="bash" >}}
+kubectl apply -n ambassador -f https://github.com/datawire/ambassador-operator/releases/latest/download/ambassador-operator-kind.yaml
+kubectl wait --timeout=180s -n ambassador --for=condition=deployed ambassadorinstallations/ambassador
+{{< /codeFromInline >}}
+
+Ambassador is now ready for use. You can try the example in [Using Ingress](#using-ingress) at this moment,
+but Ambassador will not automatically load the `Ingress` defined there. `Ingress` resources must include
+the annotation `kubernetes.io/ingress.class: ambassador` for being recognized by Ambassador (otherwise they are just ignored).
+So once the example has been loaded you can add this annotation with a `patch`:
+
+{{< codeFromInline lang="bash" >}}
+kubectl patch ingress example-ingress -p '{"metadata":{"annotations":{"kubernetes.io/ingress.class": "ambassador"}}}'
```

Comment:
nit: `kubectl annotate ingress example-ingress kubernetes.io/ingress.class=ambassador` is a bit simpler?

Reply:
Damn :facepalm: Should I change it?

Reply:
I think we should

Reply:
Fixed.